### ISSUE TITLE
Bad project name detection (?) with v0.7.1

### ISSUE THREAD

**Issue by d125q** [theleagueof/fontship#120]
When I attempt to build the master branch of https://github.com/alerque/libertinus using v0.7.1 using a local `fontship` installation, I get an error saying
```
make: *** No rule to make target '.fontship/LibertinusGitSerif-Regular-subr.otf', needed by 'LibertinusGitSerif-Regular.otf'.  Stop.
```
My guess is that the project name is inferred to be `LibertinusGit` instead of `Libertinus`.
v0.7.0 works without any problems.
(To be honest, I only suspect this to be a `fontship` issue which is why I’m reporting it here, but I’m not sure.)

**Comment by alerque** [comment]
Given that v0.7.1 was released specifically to *fix* project name detection in remote CI environments like GH Actions that reduced the amount of Git meta data that was available, it's a pretty reasonable assumption that I broke local detection in the process. I'll check it out.

One thing this highlights is that it's hart to test all the various use cases manually. See #45, but at some point I need not just one but a small suite of test repos to test local & remote runners against various project layouts and source file types (SFD, UFO, Glyphs, single vs. multiple font family per repo, etc).

Additionally the possible file layouts that *should* be detected isn't properly documented (see #119).
